chain/chunks/src/lib.rs

```diff
@@ -153,6 +168,9 @@ impl ShardsManager {
                 Duration::from_millis(CHUNK_REQUEST_SWITCH_TO_FULL_FETCH_MS),
                 Duration::from_millis(CHUNK_REQUEST_RETRY_MAX_MS),
             ),
+            seals: HashMap::new(),
+            // TODO make sure after restart we don't lose resented_by collection
```

Comment:
We don't care as much, grieving will be relatively rare, and the nodes are exected to run for quite a while, so it's OK to rebuild the reputation information after each restart

Reply:
++